packages/ts-exporter/src/main/java/org/uberfire/jsbridge/tsexporter/model/config/WebpackConfigJs.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.jsbridge.tsexporter.model.config;
+
+import org.uberfire.jsbridge.tsexporter.model.TsExporterResource;
+
+import static java.lang.String.format;
+import static org.uberfire.jsbridge.tsexporter.util.Utils.get;
+import static org.uberfire.jsbridge.tsexporter.util.Utils.lines;
+
+public class WebpackConfigJs implements TsExporterResource {
+
+    private final String npmPackageName;
+
+    public WebpackConfigJs(final String npmPackageName) {
+        this.npmPackageName = npmPackageName;
+    }
+
+    @Override
+    public String toSource() {
+
+        return format(lines(
+                "const path = require('path');",
```

Comment:
Could we have these as files in resources and parse the whole file instead of having the config pasted here like this?

Reply:
I don't think we would benefit from doing this.. I prefer to keep related things as close as possible in code.